ebean-test/src/test/java/org/tests/basic/TestLength.java

```diff
@@ -47,11 +47,13 @@ void testFileSize() throws IOException {
 
     SomeFileBean sfb2 = new SomeFileBean();
     sfb2.setContent(f1);
-    // SqlServer uses varbinary(max) and MariaDB uses a mediumblob (can not set a length), therefore the save works without exception.
+    // SqlServer uses varbinary(max) and MariaDB uses a mediumblob (can not set a length),
```

Comment:
Formatierungen darf man nicht ändern, bei Rob gibt es andere Regeln wie bei uns, bitte die nicht Code-Changes rückgängig machen

Reply:
Den Kommentar hab ich in einem PR vorher hinzugefügt.